staging/src/k8s.io/apiserver/pkg/storage/cacher/delegator_test.go

```diff
@@ -192,3 +192,55 @@ func TestCalculateDigest(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateUndelegatedListOptions(t *testing.T) {
```

Comment:
Thanks for adding this test - it addresses my concerns, because it shows that we don't fail new requests due to this change.